examples/ex1_standalone_app/build.gradle

```diff
@@ -14,5 +14,8 @@ android {
 }
 
 dependencies {
+    // External users should use the repo dependency below instead of the
+    // project dependency.
+    //compile 'com.google.android.mobly:mobly-snippet-lib:1.0.0'
```

Comment:
This line actually isn't the doc, this is the actual build file for the example.
If we change this, the examples won't be building against the lib inside the tree. This will make it hard to update the examples when we add new features.